deep_qa/models/text_classification/true_false_model.py

```diff
@@ -59,6 +59,7 @@ def _instance_type(self):
     @overrides
     def _set_max_lengths_from_model(self):
         self.max_sentence_length = self.model.get_input_shape_at(0)[1]
```

Comment:
Oh, wait.  Just `self._set_...`.  No need for the `super` there, or in other places, because you changed the method name.